packages/jaeger-ui-components/src/utils/TreeNode.ts

```diff
@@ -82,18 +88,21 @@ export default class TreeNode {
     return findPath(this, []);
   }
 
-  walk(fn, depth = 0) {
-    const nodeStack = [];
+  walk(fn: (value: string | number | undefined, node: TreeNode, depth?: number) => number | void, depth = 0) {
```

Comment:
Do we need the `fn` return type to be `number | void`? Seems like we don't care about its return type here so seems like `void` would be enough.

Reply:
Yes, this function doesn't return anything so I have set the return type to simply `void` in commit https://github.com/grafana/grafana/pull/58540/commits/8660391ac6b91981e73604f6e51c0c3b9ec39624.